hermes-common/src/main/java/pl/allegro/tech/hermes/common/message/wrapper/AvroMessageContentWrapper.java

```diff
@@ -0,0 +1,54 @@
+package pl.allegro.tech.hermes.common.message.wrapper;
+
+import org.apache.avro.io.BinaryData;
+import org.apache.avro.io.BinaryDecoder;
+import org.apache.avro.io.DecoderFactory;
+
+import java.io.IOException;
+
+public class AvroMessageContentWrapper implements MessageContentWrapper {
+
+    @Override
+    public MessageWithMetadata unwrapContent(byte[] data) {
+        BinaryDecoder binaryDecoder = DecoderFactory.get().binaryDecoder(data, null);
+
+        try {
+            return new MessageWithMetadata(
+                new MessageMetadata(binaryDecoder.readLong(), binaryDecoder.readString()),
+                binaryDecoder.readBytes(null).array());
```

Comment:
What is the point of this expression: `binaryDecoder.readBytes(null).array()`?
Can you make an empty array or something?